mautrix_telegram/__main__.py

```diff
@@ -162,6 +165,13 @@ def is_bridge_ghost(self, user_id: UserID) -> bool:
     async def count_logged_in_users(self) -> int:
         return len([user for user in User.by_tgid.values() if user.tgid])
 
+    def update_bridge_readiness(self):
+        for portal in Portal.all():
+            if portal.latest_event_timestamp and portal.latest_event_timestamp < time() - PORTAL_INACTIVE_THRESHOLD:
```

Comment:
`portal.latest_event_timestamp` will always exist on the class instance since it is being initialized there. Unless there are other portals to the one we modify?

```suggestion
            if portal.latest_event_timestamp < time() - PORTAL_INACTIVE_THRESHOLD:
```

Reply:
A `Portal` can be one of many classes.
https://github.com/vector-im/mautrix-telegram/blob/670283134bb182de6523531732fecd0c11692fec/mautrix_telegram/portal/__init__.py#L9

I only changed this attribute to `PortalTelegram`.

Reply:
No longer relevant as the implementation has changed.